pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -1091,6 +1117,20 @@ func anonymizePod(pod *corev1.Pod) *corev1.Pod {
 	return pod
 }
 
+type PodDisruptionBudgetsAnonymizer struct {
+	*policyv1beta1.PodDisruptionBudget
+}
+
+// Marshal implements serialization of a Pod with anonymization
```

Comment:
Copy/paste comment. It's just minor but is there anything we could possibly anonymize (some IDs, IP addressess and similar)?

Reply:
Yeah the comments are a mess but its WIP for a reason :D

About what to anonymize:  No idea really, the ones that I saw had nothing in them that I would consider confidential. maybe the `metadata.managedFields` can have something confidential info.

Reply:
Here is the type definition for PDB. To me it doesn't look like there could be anything worth anonymizing. What do you think ?
https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/policy/types.go

Reply:
that was my impression as well